data/data-pipeline/data_pipeline/ipython/compare_tiles_and_geoJson_files.ipynb

```diff
@@ -332,7 +331,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.8.10 ('data-pipeline-WziHKidv-py3.8')",
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
I'm a noob with notebooks, what is the significance of the display name and do we know when it's used?

Reply:
Any reason to not include the semantic versioning?